website/docs/r/video_analyzer.html.markdown

```diff
@@ -0,0 +1,132 @@
+  role_definition_name = "Reader"
+  principal_id         = azurerm_user_assigned_identity.example.principal_id
+}
+
+resource "azurerm_video_analyzer" "example" {
+  name                = "exampleanalyzer"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+
+  storage_account {
+    id          = azurerm_storage_account.example.id
+    identity_id = azurerm_user_assigned_identity.example.id
+  }
+
+  identity {
+    type = "UserAssigned"
+    identity_ids = [
+      azurerm_user_assigned_identity.example.id
+    ]
+  }
+
+  tags = {
+    environment = "staging"
+  }
+
+  depends_on = [
+    azurerm_user_assigned_identity.example,
+    azurerm_role_assignment.contributor,
+    azurerm_role_assignment.reader,
+  ]
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name of the Video Analyzer. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the resource group in which to create the Video Analyzer. Changing this forces a new resource to be created.
+
+* `location` - (Required) Specifies the supported Azure location where the resource exists. Changing this forces a new resource to be created.
+
+* `storage_account` - (Required) An `storage_account` blocks is defined below.
+
+* `identity` - (Required) An `identity` block is defined below.
+
+* `tags` - (Optional) A mapping of tags assigned to the resource.
+
+---
+
+A `storage_account` block supports the following:
+
+* `id` - (Required) Specifies the ID of the Storage Account that will be associated with the Video Analyzer instance.
+
+* `identity_id` - (Required) ID for User Assigned Managed Identity resources to be assigned.
+
+---
+
+A `identity` block supports the following:
+
+* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this Video Analyzer. Possible value is `UserAssigned`. 
+
+* `identity_ids` - (Required) A list of IDs for User Assigned Managed Identity resources to be assigned.
+
+---
+
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `id` - The ID of the Video Analyzer.
+
+## Timeouts
+
+The `timeouts` block allows you to specify [timeouts](https://www.terraform.io/docs/configuration/resources.html#timeouts) for certain actions:
+
+* `create` - (Defaults to 30 minutes) Used when creating the Video Analyzer.
+* `update` - (Defaults to 30 minutes) Used when updating the Video Analyzert.
+* `read` - (Defaults to 5 minutes) Used when retrieving the Video Analyzer.
+* `delete` - (Defaults to 30 minutes) Used when deleting the Video Analyzer.
+
+## Import
+
+Video Analyzer can be imported using the `resource id`, e.g.
+
+```shell
+terraform import azurerm_video_analyzer.analyzer /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/mygroup1/providers/Microsoft.Media/videoanalyzers/analyzer1
```

Comment:
(as above) ARM says segments should be camelCased so we should match that (since we're defining these ID's in Terraform)

```suggestion
terraform import azurerm_video_analyzer.analyzer /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/mygroup1/providers/Microsoft.Media/videoAnalyzers/analyzer1
```